components/Footer.js

```diff
@@ -8,63 +8,61 @@ import Link from "next/link"
 
 export default function Footer() {
   return (
-    <footer className="bg-gray-900 mt-12">
+    <div className="bg-gray-900 mt-12">
```

Comment:
why are we making this a **div**? If this is part of the footer, it should use the `<footer>` element

Reply:
everything in this is a div